src/nrniv/nrnpy.cpp

```diff
@@ -29,12 +29,6 @@ char* hoc_forward2back(char* s);
 // at time of configure (using the python first in the PATH).
 #if defined(NRNPYTHON_DYNAMICLOAD)
 
-#if defined(NRNCMAKE)
-// CMAKE installs libnrnpythonx.so not in <prefix>/x86_64/lib but <prefix>/lib
-#undef NRNHOSTCPU
-#define NRNHOSTCPU "."
-#endif
```

Comment:
without looking into other part of the code, In case of CMake, isn't `NRNHOSTCPU` was always `"."` but with above change thats no longer the case?

Reply:
In the compilation logs there was a warning for redefinition of `NRNHOSTCPU` so I just dropped this

Reply:
its not used in nrnpy.cpp anyway.